public/docs/advanced/configuration.md

```diff
@@ -79,6 +79,44 @@ For each network, if unspecified, transaction options will default to the follow
 * `from`: From address used during migrations. Defaults to the first available account provided by your Ethereum client.
 * `provider`: Default web3 provider using `host` and `port` options: `new Web3.providers.HttpProvider("http://<host>:<port>")`
 
+### build_directory
+
+The default output directory for compiled contracts is `./build/contracts` relative to the project root. This can be changed with the `contracts_build_directory` key.
+
+Examples:
+
+To place the built contracts in `./output/contracts`:
```

Comment:
It might be worth calling these "built contract artifacts" here

Reply:
Sure thing.